src/groups/mqb/mqbs/mqbs_filebackedstorage.h

```diff
@@ -97,7 +97,7 @@ class FileBackedStorage BSLS_KEYWORD_FINAL : public ReplicatedStorage {
     // Most likely number of records for each guid (one each of message,
     // confirm & deletion record).  This number is correct for every queue
     // except for the fanout one, which has more than 1 confirm records.
```

Comment:
```suggestion
    // The most probable number of records for each guid for priority queue.
    // Currently, the value is 2: one data record + one deletion record.
    // With last confirm optimization, we don't write a last confirm, and don't
    // count it here.
    // For fanout queues, the expected number of records is more than this:
    // one data record + (number of appIDs - 1) confirms + one deletion record,
    // where -1 due to last confirm optimization.
```
I would change the comment above, since it's outdated.